Groups

IngCr3at1on opened the pull request.

IngCr3at1on committed:
Implement minor group functionality

IngCr3at1on committed:
Add some convenience functions for finding a specific client object

IngCr3at1on committed:
block users from sending messages from outside a group

this effects joining a room as well

GrimTheReaper commented:
Approved.

Nice use of data structure with Redis.

[![Approved with PullApprove](https://img.shields.io/badge/pullapprove-approved-brightgreen.svg)](https://pullapprove.com/JustAnotherOrganization/tiberious/pull-request/33/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=JustAnotherOrganization/tiberious)

IngCr3at1on committed:
Fix bug on init and add some minor tests for roomhandler

The #default group wasn't being initialized during init without a database
being enabled.

IngCr3at1on commented:
Thanks, I totally just added another commit though after your approval :P
